extensions/espanso/package.json

```diff
@@ -6,14 +6,24 @@
   "icon": "espanso-logo.png",
   "author": "kud",
   "contributors": [
-    "nbbaier"
+    "nbbaier",
+    "erics118"
   ],
   "categories": [
     "Applications",
     "System",
     "Productivity"
   ],
   "license": "MIT",
+  "preferences": [
+    {
+      "name": "espansoPath",
+      "type": "textfield",
```

Comment:
```suggestion
      "type": "directory",
```

Reply:
The proper type would be `file`, not directory here. But, I think it might be better to keep it as a textfield because this is what a lot of other extensions currently do, eg toothpick, bitwarden, 1password, yt downloader, etc

Reply:
yeah I was not sure either about this, since it's a file. Let's go with `textfield`.